src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,31 @@
+/* eslint-disable react/jsx-fragments */
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <React.Fragment key={movie.imdbId}>
+        <MovieCard
+          title={movie.title}
+          description={movie.description}
+          imgUrl={movie.imgUrl}
+          imdbUrl={movie.imdbUrl}
+        />
+      </React.Fragment>
+    )) }
+  </div>
 );
+
+interface Movie {
```

Comment:
It'll be better if you transfer this interface into seperate file and import it in this file. In this case it's also correct, because you have only one interface, but if you need more you should create file with types)

Reply:
I agree with you. Fixed